cmds/nspawn

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+
+nspawn_list() {
+    local path=${1}
+
+    ls -1 "${path}"
+}
+
+nspawn_create() {
+    local path=${1}
```

Comment:
I have converted these into scripts with some documentation a while back: https://github.com/eric-ch/bordel/tree/systemd-nspawn-helpers/systemd-nspawn.
Might be worth re-working and try to do factorize the nspawn containers a bit to instead compose the installed bits in a base container.
... Also the versions in the script names are wrong.

Reply:
Your scripts and documentation look good. I didn't know about them.

Do you just ssh into that container and use bordel locally?  I think that's what I'd move toward to get the nicest shell experience. Downside of this command is only a single session can "enter" the container at a time.

Reply:
Yes, I do stuff on the host to use my environment configuration and and use ssh to build in the container using bordel/bitbake/others. That being said one does not prevent the other, and the single session "enter" might be preferable other people.